Power Up Proto/src/org/usfirst/frc/team5493/robot/subsystems/DriveBase.java

```diff
@@ -32,23 +33,30 @@
 	public DriveBase() {
 		super();
 
-		// leftFrontMotor = new WPI_TalonSRX(RobotMap.LEFT_FRONT);
-		// leftBackMotor = new WPI_TalonSRX(RobotMap.LEFT_BACK);
-		// rightFrontMotor = new WPI_TalonSRX(RobotMap.RIGHT_FRONT);
-		// rightBackMotor = new WPI_TalonSRX(RobotMap.RIGHT_BACK);
+		// TODO: This should be set back... the other code (lines 42 and 43 was test
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)